Product/SoapUI_Test/RegressionSuite/Manual/RequestResponseContentTypeTest/pom.xml

```diff
@@ -0,0 +1,64 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <groupId>org.connectopensource</groupId>
+        <artifactId>Manual</artifactId>
+        <version>4.7.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>RequestResponseContentTypeTest</artifactId>
+    <name>Maven 3 soapUI RequestResponseContentTypeTest</name>
+    <packaging>jar</packaging>
+    <modelVersion>4.0.0</modelVersion>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>com.github.redfish4ktc.soapui</groupId>
+                <artifactId>maven-soapui-extension-plugin</artifactId>
+                <configuration>
+                    <settingsFile>${project.basedir}/../../src/main/resources/soapui-settings.xml</settingsFile>
+                    <outputFolder>${project.build.directory}/soapui/output/</outputFolder>
+                    <soapuiProperties>
+                        <property>
+                            <name>soapui.logroot</name>
+                            <value>${project.build.directory}/soapui/my_custom_logs/</value>
+                        </property>
+                        <property>
+                            <name>soapui.scripting.library</name>
+                            <value>${project.basedir}/../../src/main/resources/groovy_scripts</value>
+                        </property>
+                        <property>
+                            <name>soapui.ext.libraries</name>
+                            <value>${project.basedir}/../../src/main/resources/ext</value>
+                        </property>
+                    </soapuiProperties>
+                    <testSuite>${test.suite}</testSuite>
+                    <testCase>${test.case}</testCase>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>PatientDiscovery</id>
+                        <phase>integration-test</phase>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <configuration>
+                            <projectFile>${project.basedir}/PatientDiscovery.xml</projectFile>
```

Comment:
Inside this folder "RequestResponseContentTypeTest", Added execution block  for all soap Projects in pom.xml.